README.md

```diff
@@ -1 +1,156 @@
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+- <summary>Client</summary>
+    <ul>
+      <li><a href="https://reactjs.org/">HTML</a></li>
+      <li><a href="https://reactjs.org/">CSS</a></li>
+      <li><a href="https://reactjs.org/">JavaScript</a></li>
+      <li><a href="https://reactjs.org/">ReactJS</a></li>
+      <li><a href="https://reactjs.org/">API</a></li>
+    </ul>
+
+### Key Features <a name="key-features"></a>
+
+- **[Responsive layout]**
+- **[UX/UI accessibility]**
+- **[Dynamic data]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps:
+
+### Prerequisites
+
+In order to run this project you need:
+
+- A browser of you choice.
+- A text editor of your choice.
+- An installed node.js on your local system
+
+### Setup
+
+Clone this repository to your desired folder:
+
+- Use the following Commands:
+
+      cd your-desired-folder
+      git@github.com:riskmaris/math-magicians.git
+
+### Install
+
+- Install this project with:
+
+       npm react-app .
+       npm start
+
+### Usage
+
+- Use following commands to run on your local system:
+
+      npm run build
+      npm run deploy
+
+### Run tests
+
+- Run the following script and style test:
+
+      npx eslint "**/*.{js,jsx}"
+      npx eslint "**/*.{js,jsx}" --fix
+
+      npx stylelint "**/*.{css,scss}"
+      npx stylelint "**/*.{css,scss}" --fix
+
+### Deployment
+
+- You can deploy this project using:
+
+      npm run build
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Zelalem Yohannes**
+
+- GitHub: [Zelalem Yohannes](https://github.com/Zel-hub7)
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **[Add About page]**
+- [ ] **[Add Contact page]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/riskmaris/math-magicians).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project, give it a star.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank Microverse and my coding partners.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ❓ FAQ (OPTIONAL) <a name="faq"></a>
+
+- **How to make it mobile friendly?**
+
+  - Put a viewport tag in the header
+
+- **How to design the site?**
+
+  - Draw a mockup before you start to code
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] MIT License is not displaying as intended. Kindly note that your license file is MIT.md which does not correspond to the link provided in the ReadMe.

**Required file not found:**
![image](https://github.com/Zel-hub7/Math-magicians/assets/12745474/c8972c00-d8a4-4bcd-9d72-f8d6250df1ef)